tests/integration/adapters/adapter-test.js

```diff
@@ -180,3 +180,24 @@ test('it sets meta in included records', function(assert) {
     assert.equal(this.get('metadata').read({ id: 2, type: 'reference' }).get('something'), 20);
   });
 });
+
+test('it supports empty responses', function(assert) {
+  const EMPTY_RESPONSE = '';
+
+  answers.push({
+    data: {
+      type: 'examples',
+      id: 1
+    }
+  });
+
+  answers.push(EMPTY_RESPONSE);
+
+  return RSVP.resolve().then(() => {
```

Comment:
Not sure why the `RSVP.resolve` is needed here, just copy pasted it from the other tests.